src/Compilers/VisualBasic/Portable/Binding/Binder_Expressions.vb

```diff
@@ -694,9 +694,9 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
                      SyntaxKind.GenericName
                     value = DirectCast(node.Argument, SimpleNameSyntax).Identifier.ValueText
 
-                Case Else
+                    ' Case Else
```

Comment:
>' Case Else [](start = 20, length = 11)

Why is the code in the compiler commented out?

Reply:
When calling this function (indirectly) by the codefix: 

https://github.com/dotnet/roslyn/blob/master/src/Workspaces/VisualBasic/Portable/CodeGeneration/VisualBasicSyntaxGenerator.vb#L102

This assert is triggered in the compiler. Since this is likely a bug in the syntax generator I told them to comment this out so they were not blocked. We will need to fix the underlying issue here before merging this in.

Its possible that this issue has existed for a long time since analyzers and codefixes that generate nameof expressions are almost always being loaded and run against the release compiler, where this assert is not fired.